srcpkgs/ardour/template

```diff
@@ -31,6 +29,14 @@ case "$XBPS_TARGET_MACHINE" in
 	*)        broken="Don't know build target" ;;
 esac
 
+# Upstream deletes older release tarballs from their site and
+# GitHub tarballs are empty: https://tracker.ardour.org/view.php?id=7328
+do_fetch() {
+	git clone git://github.com/Ardour/ardour ${pkgname}-${version}
+	cd ${pkgname}-${version}
```

Comment:
You can use `${wrksrc}` instead of `${pkgname}-${version}`. Then it's good to merge.

Reply:
done :)